.github/config/typos.toml

```
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[files]
+extend-exclude = [
+    "src/common/sha1.cc",
+    "tests/gocase/util/slot.go",
+]
```

Very nice to see Typos in action!

```suggestion
[files]
extend-exclude = [
     ".git/",
    "src/common/sha1.cc",
    "tests/gocase/util/slot.go",
]
ignore-hidden = false
```

This checks dotfiles too.

---

Thank you!

---

Glad to contribute.